clients/camunda-spring-boot-starter/src/main/java/io/camunda/client/spring/configuration/CamundaAutoConfiguration.java

```diff
@@ -23,7 +23,7 @@
 import org.springframework.boot.autoconfigure.AutoConfigureAfter;
 import org.springframework.boot.autoconfigure.ImportAutoConfiguration;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
-import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.boot.jackson.autoconfigure.JacksonAutoConfiguration;
```

Comment:
based on feedback and discussion with @jonathanlukas , we decided to drop this dependency and decouple the setup from jackson autoconfig.

In the past it lead to the starter reusing a jackson mapper setup for the spring web layer by default, we see no reason to keep that but would rather decouple this starter from the jackson autoconfig. In case users already declared their own mappers explicitly before, these will be picked up still, so this should not be a breaking change for those who intended to do that.